restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/Parser.java

```diff
@@ -118,11 +123,16 @@ private void start(final Stack<Tag.Builder> builders, final XMLStreamReader stre
     }
 
     private Tag next() throws LimitExceededException{
+		boolean shouldContinue; // new variable
```

Comment:
maybe rename var to "attsValidated" or something that means the attrbiutes were correctly validated